src/components/collab/show/CollabDisplay.js

```diff
@@ -88,13 +89,9 @@ export const CollabDisplay = () => {
       const metadataFile = res[0] ? res[0].metadata_file : false
 
       if (metadataFile) {
-        axios
-          .get(
-            `https://cloudflare-ipfs.com/ipfs/${metadataFile.split('//')[1]}`
-          )
-          .then(({ data }) => {
-            setLogo(data.identicon)
-          })
+        axios.get(HashToURL(metadataFile)).then(({ data }) => {
```

Comment:
i think it would be better to use `await` here, otherwise errors won't be propagated to the `catch()` function that comes after `fetchMeta()`